sys/net/transport_layer/ng_udp/ng_udp_hdr_print.c

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (C) 2015 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     net_pktdump
+ * @{
+ *
+ * @file
+ * @brief       Generic module to dump packages received via netapi to STDOUT
+ *
+ * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>
+ *
+ * @}
+ */
+
+#include <stdio.h>
+
+#include "net/ng_udp.h"
+
+void ng_udp_hdr_print(ng_udp_hdr_t *hdr)
+{
+    printf("   src-port: %5" PRIu16 "  dst-port: %5" PRIu16 "\n",
+           byteorder_ntohs(hdr->src_port), byteorder_ntohs(hdr->dst_port));
+    printf("   length: %" PRIu16 "\n", byteorder_ntohs(hdr->length));
```

Comment:
Because I figured, that a wrong checksum will lead to errors that are outputted on other levels, so I dropped it here... But I can add it.